src/app/lib/config/services/arabic.js

```diff
@@ -124,17 +124,17 @@ export const service = {
           description: {
             uk: {
               first: 'نستخدم ',
-              linkText: 'ملفات ارتباط (كوكيز)',
+              linkText: 'ملفات ارتباط',
               last:
-                ' لمنحك أفضل خدمة رقمية. الرجاء أحطنا علما إذا كنت توافق على تحميل كل هذه الكوكيز.',
+                ' لمنحك أفضل خدمة رقمية. الرجاء أحطنا علما إذا كنت توافق على تحميل كل هذه الملفات cookies .',
               linkUrl:
                 'https://www.bbc.co.uk/usingthebbc/cookies/what-do-i-need-to-know-about-cookies/',
             },
             international: {
               first: 'نستخدم نحن وشركاؤنا تقنيات مثل ',
-              linkText: 'الكوكيز',
+              linkText: 'ملفات الارتباط',
               last:
-                '، كما نقوم بجمع معلومات خاصة بالتصفح من أجل توفير أفضل خدمة رقمية ولاضفاء مسحة شخصية للمحتوى والاعلانات الموجهة إليك. الرجاء إعلامنا إذا كنت موافقا على ذلك.',
+                '، كما نقوم بجمع معلومات خاصة بالتصفح من أجل توفير أفضل خدمة رقمية ولجعل المحتوى والاعلانات، الموجهة إليك، شخصية. الرجاء إعلامنا ',
```

Comment:
I'm not sure this one is correct. In the translations spreadsheet I can see the following:

```suggestion
                '، كما نقوم بجمع معلومات خاصة بالتصفح من أجل توفير أفضل خدمة رقمية ولجعل المحتوى والاعلانات، الموجهة إليك، شخصية. الرجاء إعلامنا إذا كنت موافقا على ذلك.',
```